### PR TITLE
Event scroll new js rb

### PR DESCRIPTION
**Pull request by dlangston**
screen shot 2015-08-25 at 12 26 20 pm

### CODE REVIEW

File: ooiui/templates/science/plotting.html [hunk 1]
```diff
@@ -36,100 +35,104 @@
...
+        <div class="new-heading">
+          Graph
+        </div>
+
+        <div class="panel-body">
+          <!--  Plot section NORMAL   -->
+          <div id="plot-view">
+            {# <i class="fa fa-bar-chart" style="margin-left:50%;font-size:90px;"> </i>#}
+            {# <img src="/img/graph7.png" style="width:100%;"> #}
+            <img src="/img/plotting/graph.png" style="width:100%;">
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @dlangston move style to css

File: ooiui/templates/science/plotting.html [hunk 2]
```diff
@@ -36,100 +35,104 @@
...
+          Events
+        </div> <!-- panel-heading -->
+
+        <div class="scrollable-block">
+          <div class="scrollable-area">
+            <div class="panel-body">
+              <div id="events-table">
+                {# <i class="fa fa-list-alt" style="margin-left:50%;font-size:90px;"> </i>                 #}
+                {# <img src="/img/cal.jpg" style="width:1120px; height:90px;">	       	        #}
+                <img src="/img/plotting/events.png" style="width:100%;">
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @dlangston move style to css

File: ooiui/templates/science/plotting.html [hunk 3]
```diff
@@ -36,100 +35,104 @@
...
+
+    <div class="row">
+      <div id='annotation-panel' class="panel">
+        <div class="new-heading">
+          Annotations
+        </div>
+        <div class="panel-body">
+          <div id="annotation-table">
+            {# <i class="fa fa-th-list" style="margin-left:50%;font-size:90px;"> </i>                 #}
+            <img src="/img/plotting/anotations.png" style="width:100%;">
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @dlangston move style to css

File: ooiui/templates/science/plotting.html [hunk 4]
```diff
@@ -462,73 +465,83 @@
     //  
     this.listenTo(this.collections.plotEventList, 'reset', function(collection) {
       self.views.plotEventListView.collection = collection;
+
+
+
       $.when(self.views.plotEventListView.render()).done(function() {
-        $('#eventsTable').fixedHeaderTable({ altClass: 'odd', footer: true, fixedColumns: false, });  
+
+        $("table").stickyTableHeaders({ scrollableArea: $(".scrollable-area"), "fixedOffset": 2 });
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @dlangston no id? were just grabbing the table, im thinking this should have an id...

File: ooiui/templates/science/plotting.html [hunk 5]
```diff
@@ -462,73 +465,83 @@
...
+
+        console.log("made TI");
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @dlangston remove console log

**Comment by birdage** [comment]
@dlangston check you are using spaces not tabs in the editor

**Comment by birdage** [comment]
@dlangston i also suggest squashing the commits down to just 1.

**Comment by DanielJMaher** [comment]
@dlangston Merge COnflict.  Fix.

**Comment by birdage** [comment]
@dlangston can we close this now?

**Comment by dlangston** [comment]
I thought I closed this? Did @DanielJMaher  try and merge this after it was closed?

**Pull request closed without merging**